lib/open_project/configuration.rb

```diff
@@ -191,6 +191,8 @@ def configure_cache(application_config)
           cache_config = [:dalli_store]
           cache_config << @config['cache_memcache_server'] \
             if @config['cache_memcache_server']
+        elsif cache_store == :file_store
+          cache_config = [:file_store, Rails.root.join("tmp/cache/store")]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.